typedefs/gqlTypes.ts

```diff
@@ -0,0 +1,387 @@
+import { GraphQLResolveInfo } from 'graphql'
```

Comment:
What do you think about not checking in generated code into the repo?

Reply:
we need references to it otherwise we wouldn't be able to build properly. 
I don't think requiring generation per person would bring any benefit. 🤔 
Basically, if we have an updated schema, someone would just run it again and check it in so everyone has it. Also, it would allow us to version and only update the client-side schema when we know it works.